qiskit_optimization/applications/max_cut.py

```diff
@@ -117,7 +117,7 @@ def parse_gset_format(filename: str) -> np.ndarray:
                     s__, t__, _ = v
                     s__ -= 1  # adjust 1-index
                     t__ -= 1  # ditto
-                    w[s__, t__] = t__
```

Comment:
Since `_` is usually used for non-used value, I suggest renaming it, e.g., `w__`.
```suggestion
                    s__, t__, w__ = v
                    s__ -= 1  # adjust 1-index
                    t__ -= 1  # ditto
                    w[s__, t__] = w__
```

Reply:
Thanks! I changed it as you proposed.

Reply:
My suggestion does not seem to be applied.
Line 120 is stil
```
                    w[s__, t__] = _
```